ogame/__init__.py

```diff
@@ -336,6 +337,27 @@ def inConstruction(self):
         else:
             return False
 
+    def techtree(self, id):
+        id = id(0)[0]
+        dico_tree = {}
+        r = self.session.get(
+            self.index_php + f'page=ajax&component=technologytree&ajax=1&technologyId={id}&tab=1'
+        ).text
+        bs3 = BeautifulSoup4(r)
+        i = bs3.find('div', {'class': 'content technologytree'}).find_all('div', {'class': 'techWrapper'})
```

Comment:
Yeah it makes it hard to understand.
i is normaly a variable i assosiate with index and not an bs class

Reply:
Oh ? Do you have an example ?

Reply:
call it what it is.
in this line your searching technologys.
tech, technology, techs, technologys, technology_tree, tree, tech_nodes, requirements, technology_requirement.

Reply:
so... it's just the link to grap the treetech, it's the same for the shipyard or another items...